### ISSUE TITLE
2.0: Does not work with sphinx 1.4.9?

### ISSUE THREAD

**Issue by olebole** [astropy/astropy-helpers#325]
Hi,

after uploading 2.0, I got a bug report [debian/#867772](https://bugs.debian.org/867772) that python-drizzle (from STSCI) [could not be built](https://tests.reproducible-builds.org/debian/rbuild/unstable/amd64/python-drizzle_1.5-2.rbuild.log).
```
Traceback (most recent call last):
  File "setup.py", line 123, in <module>
    **package_info
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/usr/lib/python2.7/dist-packages/astropy_helpers/commands/build_sphinx.py", line 260, in run
    AstropyBuildDocs.run(self)
  File "/usr/lib/python2.7/dist-packages/astropy_helpers/commands/build_sphinx.py", line 170, in run
    """).format(pdb=self.pdb, verbosity=self.verbosity,
  File "/usr/lib/python2.7/distutils/cmd.py", line 105, in __getattr__
    raise AttributeError, attr
AttributeError: pdb
```
That was with sphinx 1.4.9. Now, when Debian has upgraded to sphinx 1.5.6, the problem disappeared.
Just for completeness (and possible backporting): Is sphinx-1.4 support silently dropped?

**Comment by bsipocz** [comment]
It may be a side effect of one of the PRs, let me dig into it.

**Comment by bsipocz** [comment]
this is closed by #326
